sensor_msgs/Image image
---
detic_ros_emia/SegmentationInfo seg_info
sensor_msgs/Image debug_image
